dffml/df/archive.py

```diff
@@ -0,0 +1,214 @@
+import uuid
+import pathlib
+import zipfile
+import tarfile
+import mimetypes
+
+from typing import Dict, Tuple, Any
+from .types import DataFlow, Input, InputFlow, Definition, Operation
+from ..operation.archive import (
+    make_tar_archive,
+    make_zip_archive,
+    extract_tar_archive,
+    extract_zip_archive,
+)
+from ..operation.compression import (
+    gz_compress,
+    gz_decompress,
+    bz2_compress,
+    bz2_decompress,
+    xz_compress,
+    xz_decompress,
+)
+
+
+def get_key_substr(string: str, dict: dict, return_="value") -> Any:
```

Comment:
```suggestion
def get_key_substr(string: str, dict: dict, return_value: bool = True) -> Any:
```

Reply:
Have made this change is a local commit would push it soon .... :)

Reply:
Done 👍🏽